src/router.ts

```diff
@@ -247,6 +247,7 @@ function createQueryRoute({
   };
 
   router.route({
+    operationId: fieldName,
```

Comment:
I'm not sure `fieldName` is a good idea since different root types can have a field with the same name. The name of the operation looks better fit in this case.